signin/signin.js

```diff
@@ -0,0 +1,100 @@
+//focusout 이벤트 리스너
+document
+  .getElementById("email-input")
+  .addEventListener("focusout", function () {
+    let email = this.value;
```

Comment:
this보다는 focusout에서 주어지는 event 객체의  target을 사용하면 어떨까요?
```suggestion
  .addEventListener("focusout", function (event) {
    let email = event.target.value;
```